src/testing/platform/testing-build.ts

```diff
@@ -2,7 +2,7 @@ import type * as d from '@stencil/core/internal';
 
 export const Build: d.UserBuildConditionals = {
   isDev: true,
-  isBrowser: false,
-  isServer: true,
+  isBrowser: true,
+  isServer: false,
```

Comment:
Needed to make these changes for unit tests leveraging `forceUpdate` to pass. No other tests started to fail with this change and the `newSpecPage` helper uses the lazy build which would traditionally have `isBrowser: true`. So, I believe this is a low-risk change